llarp/router/rc_lookup_handler.cpp

```diff
@@ -279,7 +279,8 @@ namespace llarp
 
       if(lookupRouters.size() > LookupPerTick)
       {
-        static std::mt19937_64 rng{std::random_device{}()};
+        uint64_t seed = llarp::randint();
+        static std::mt19937_64 rng{seed};
```

Comment:
This should be inlined:
```c++
static std::mt19937_64 rng{llarp::randint()};
```
so that it only calls randint() once (the first time the mt19937_64 is initialized).

Reply:
ah ok